### ISSUE TITLE
Add method to fetch the history and comments for a workitem

### ISSUE THREAD

**Issue by vishal-kharde** [dixudx/rtcclient#122]

**Comment by fignutz** [comment]
Currently,  workitem History is NOT reported in the OSLC API, but in the newer Reportable REST API that IBM uses. That said, I was able to hack this library set to pull in item histories, but it is very involved and requires, amongst writing your own algorithm to sort the history return object (_not the same as an OSLC object at all_), creating a custom `relogin` function to "switch headers" on the fly within your RTC Class object: esentiually bootstrapping your previous login with the new headers to talk to the other REST Service. 

In other words, the headers that come pre-packed with this library are incompatible with the RPT/Reportable API header. I doubt that the author (@dixudx ) will add this in anytime soon.

**Comment by dixudx** [comment]
@fignutz Currently I don't have too much bandwidth to work on this new feature. Your contributions are welcome if you are interested.
